app/domain/sac_imports/wso2_person_entry.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacImports
+  class Wso2PersonEntry
+    GENDERS = {HERR: "m", FRAU: "w"}.freeze
+    LANGUAGES = {D: "de", F: "fr", I: "it", E: "en"}.freeze
+    DEFAULT_LANGUAGE = "de"
+    DEFAULT_COUNTRY = "CH"
+
+    include ActiveModel::Validations
+
+    validate :existing_and_wso2_email_matches
+    validate :valid_gender
+    validate :at_least_one_role
+    validate :person_must_exist_if_navision_id_is_present
+
+    def existing_and_wso2_email_matches
+      if person.persisted? && row[:email] != person.email
+        errors.add(:email, "#{row[:email]} does not match the current email")
```

Comment:
We need to normalize the email in `row[:email]` since the email in person is normalized.